### PR TITLE
Define styling for contacts.

### PR DESCRIPTION
**Pull request by bierik**
:warning: Depends on https://github.com/4teamwork/ftw.theming/pull/46

screencapture-localhost-8080-plone-contacts-1458556411086
screencapture-localhost-8080-plone-contacts-1458556466044
screencapture-localhost-8080-plone-contacts-1458556482528
screencapture-localhost-8080-plone-contacts-donec-vitae-nullam-dictum-1458556529318
bildschirmfoto 2016-03-21 um 11 33 48
bildschirmfoto 2016-03-21 um 11 33 58
bildschirmfoto 2016-03-21 um 11 34 06
bildschirmfoto 2016-03-21 um 11 35 09
bildschirmfoto 2016-03-21 um 11 40 09
bildschirmfoto 2016-03-21 um 11 40 21
bildschirmfoto 2016-03-21 um 11 40 41
bildschirmfoto 2016-03-21 um 11 40 54

### CODE REVIEW

File: ftw/contacts/browser/contact.py [hunk 1]
```diff
@@ -35,6 +35,18 @@ def get_address_map(self):
 
         return address_map
 
+    @property
+    def portait_css_class(self):
```
Comment (elioschmutz) [inline ftw/contacts/browser/contact.py]: typo

**Comment by bierik** [comment]
@elioschmutz updated

**Comment by elioschmutz** [comment]
@bierik I get an error:

``` bash
SassError: Error parsing block:
    @include cell();
@include gridposition($x: 0, $offset: false);
@include gridwidth($x: 16, $offset: false);
display: block;

on line 73 of /Users/elioschmutz/projects/ses.web/src/ftw.contacts/ftw/contacts/resources/scss/search.scss
Traceback:
  File "/Users/elioschmutz/Plone/eggs/pyScss-1.3.4-py2.7.egg/scss/compiler.py", line 374, in manage_children
    self._manage_children_impl(rule, scope)
  File "/Users/elioschmutz/Plone/eggs/pyScss-1.3.4-py2.7.egg/scss/compiler.py", line 403, in _manage_children_impl
    method(calculator, rule, scope, block)
  File "/Users/elioschmutz/Plone/eggs/pyScss-1.3.4-py2.7.egg/scss/compiler.py", line 777, in _at_include
    callee_namespace, mixin, args, kwargs)
  File "/Users/elioschmutz/Plone/eggs/pyScss-1.3.4-py2.7.egg/scss/compiler.py", line 640, in _populate_namespace_from_call
    raise NameError("%s has no such argument %s" % (name, kwargs.keys()[0]))
NameError: Mixin gridposition has no such argument $x
```

**Comment by elioschmutz** [comment]
@bierik I get an error:

``2016-03-24 14:26:41 ERROR scss.compiler Mixin not found: contact-portrait:0 (/Users/elioschmutz/projects/ses.web/src/ftw.contacts/ftw/contacts/resources/scss/contacts.scss:11)``

**Comment by bierik** [comment]
Do you have checked out this hash for ftw.theming? https://github.com/4teamwork/ftw.theming/pull/46

**Comment by elioschmutz** [comment]
:+1:

**Comment by elioschmutz** [comment]
Tests are passing. They where failing because an issue in ftw.geo on Plone 4.3.8 which is solved now (https://github.com/4teamwork/ftw.geo/pull/18)

**Pull request merged**